.github/workflows/release.yml

```diff
@@ -0,0 +1,47 @@
+name: Release
+
+permissions:
+  contents: write
+
+on:
+  push:
+    tags:
+      - "v*"
+
+jobs:
+  Release:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 16.x
+
+      - uses: actions/setup-python@v4
+        with:
+          python-version: "3.x"
+
+      - run: |
+          python -m pip install --upgrade pip
+          pip install build
+          python -m build
+
+      - name: Publish distribution 📦 to PyPI
+        uses: pypa/gh-action-pypi-publish@release/v1
+        with:
+          password: ${{ secrets.PYPI_API_TOKEN }}
+      
+      - name: Publish distribution 📦 to NPM
+        run: |
+          yarn install
+          yarn build
+          yarn publish
+        env:
+          NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
```

Comment:
I want to publish a JS library that generates several Gosling specs (e.g., `createOverviewSpec()`, `createSvSpec()`, etc) so that it can be used in the Python package to visualize individual views.

I am unsure if it is fine to publish Python and JS packages at the same time.